scopes/react/react/templates/react-workspace/files/workspace-config.ts

```diff
@@ -1,24 +1,29 @@
 import { WorkspaceContext } from '@teambit/generator';
-import { getWorkspaceConfigTemplateParsed, stringifyWorkspaceConfig } from '@teambit/config';
 
-export async function workspaceConfig({ name, defaultScope }: WorkspaceContext) {
-  const configParsed = await getWorkspaceConfigTemplateParsed();
-  configParsed['teambit.workspace/workspace'].name = name;
-  configParsed['teambit.workspace/workspace'].defaultScope = defaultScope || 'my-scope';
-  configParsed['teambit.dependencies/dependency-resolver'].packageManager = 'teambit.dependencies/pnpm';
-  configParsed['teambit.dependencies/dependency-resolver'].policy = {
-    dependencies: {},
-    peerDependencies: {
-      '@testing-library/react': '11.2.6',
-      react: '16.13.1',
-      'react-dom': '16.13.1',
+export function workspaceConfig({ name, defaultScope }: WorkspaceContext) {
```

Comment:
@debs-obrien , it's actually better not to paste the content of the `workspace.jsonc` here. The way how it was is by re-using the template in the core, and it has two advantages:
1. if we use the core template, it'll be changed here.
2. the comments are preserved. (this was asked by Cody. it was confusing when the comments were missing).

Reply:
ok great. change made